scripts/pylib/twister/twisterlib/testsuite.py

```diff
@@ -450,7 +450,7 @@ def get_unique(testsuite_root, workdir, name):
             relative_ts_root = ""
 
         # workdir can be "."
-        unique = os.path.normpath(os.path.join(relative_ts_root, workdir, name))
+        unique = os.path.normpath(os.path.join(relative_ts_root, workdir, name)).replace(os.sep, '/')
```

Comment:
nit: with that change the rest shouldn't be required, because you'll never get `\\` in testplan. 
Hope you tested it well and checked, if `\\` is not required somewhere, when you run twister on Windows.

Reply:
OK